src/ngx_http_lua_worker.c

```diff
@@ -130,4 +130,27 @@ ngx_http_lua_ffi_worker_count(void)
 
     return (int) ccf->worker_processes;
 }
+
+
+int
+ngx_http_lua_ffi_worker_type(void)
+{
+    return ngx_process;
+}
+
+
+void
+ngx_http_lua_ffi_worker_privileged(int enable)
```

Comment:
I finally decided to use `ngx_http_lua_ffi_privileged_agent`, I think the user can choose to enable or disable, free to switch.